tests/parameters.yaml

```diff
@@ -0,0 +1,9 @@
+parameters:
+    application_cache:
+        adapter: apcu
+        options:
+            namespace: 'app_tests'
+    translation_cache:
+        adapter: apcu
```

Comment:
probably not a good idea to rely on apcu for translations as far as I remember. Probably better to use fs (as in travis config file)